### PR TITLE
Ldn8 - JS1-W1 - Saman Zahedi

### PR DESCRIPTION
**Pull request by SamanZahedi**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?
@beckyf0keefe Please review my work 

-----
[View rendered exercises/C-variables/README.md](https://github.com/SamanZahedi/JavaScript-Core-1-Coursework-Week1/blob/Ldn8/JS1/saman-zahedi/exercises/C-variables/README.md)
[View rendered exercises/G-numbers/README.md](https://github.com/SamanZahedi/JavaScript-Core-1-Coursework-Week1/blob/Ldn8/JS1/saman-zahedi/exercises/G-numbers/README.md)

### CODE REVIEW

File: exercises/E-strings-concatenation/exercise.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+let greetingStart = "Hello, my name is";
+let name = "Saman";
+console.log(`${greetingStart} ${name}`);
```
Comment (mandeepsangha) [inline exercises/E-strings-concatenation/exercise.js:4]: good use of Concat

File: mandatory/4-tax.js [hunk 2]
```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price + (price * 20) / 100;
```
Comment (mandeepsangha) [inline mandatory/4-tax.js:9]: another way of achieving this is return price * 1.2 ;

**Review by mandeepsangha — COMMENTED** [review]
Hi Saman, i have reviewed your code, very well done

File: extra/1-currency-conversion.js [hunk 3]
```diff
@@ -5,8 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
-
+function convertToUSD(price) {
+  return price * 1.4;
```
Comment (DavoodKhoshnood) [inline extra/1-currency-conversion.js:9]: Saman jan! you should use .toFixed(2) to show 2 decimal number

File: extra/1-currency-conversion.js [hunk 4]
```diff
@@ -15,7 +16,9 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  return ((price - price / 100) * 5.7).toFixed(2);
```
Comment (DavoodKhoshnood) [inline extra/1-currency-conversion.js:20]: Saman jan! you didn't use 99% of price. you can use something same as this:
(price * 99) / 100

**Pull request closed without merging**